### DISCUSSION TITLE
Skip permission check of directory for decreasing invocation of getattr ?

### DISCUSSION THREAD

**Discussion by carson-yang** [libfuse/libfuse#841] [Q&A] [answered]
I want to improve the performance of reading and writing small file,  I hope to just do permission check for file not directory.
Is this possible?

**Comment by trapexit** [comment, marked as answer]
Not sure exactly what you are asking. Unless you enable the default permissions flag when mounting your filesystem you are entirely responsible for dealing with permissions.
